File: MODUL_4/C/PRAK402_2310817310014_MUHAMMADRIZKISAPUTRA.c
#include <stdio.h>

int main() {
    int N;
    printf("Masukkan bilangan: ");
    scanf("%d", &N);

    for (int i = 1; i <= N; i += 2) {
        printf("%d ", i);
    }
    printf("\n");

    for (int i = (N % 2 == 0) ? N : N - 1; i >= 2; i -= 2) {
        printf("%d ", i);
    }
    printf("\n");

    return 0;
}

File: MODUL_4/Py/PRAK402_2310817310014_MUHAMMADRIZKISAPUTRA.py
N = int(input("Masukkan bilangan: "))

for i in range(1, N+1, 2):
    print(i, end=" ")
print()

for i in range(N if N % 2 == 0 else N - 1, 1, -2):
    print(i, end=" ")
print()
